EVSL_1.0/SRC/landosG.c

```diff
@@ -42,51 +44,23 @@
  *    landos.c/LanDos is only for the standard eigenvalue problem.
  *----------------------------------------------------------------------*/
 
-int LanDosG(const int nvec, const int msteps, const int degB, int npts,
-            double *xdos, double *ydos, double *neig, const double *const intv,
-            const double tau) {
+int LanDosG(const int nvec, const int msteps, int npts, double *xdos, double *ydos, 
+            double *neig, const double *const intv) {
 
+  int i, j, k;
   int maxit = msteps, m;  /* Max number of iterations */
   int n = evsldata.n; /* Number of elements in matrix */
-
-
   const int ifGenEv = evsldata.ifGenEv;
-
-  polparams pol_sqr;
-  polparams pol_sol;
-
-  if (ifGenEv) {
-    set_pol_def(&pol_sqr);
-    set_pol_def(&pol_sol);
-
-    double *mu_sqr;
-    double *mu_sol;
-
-    Calloc(mu_sqr, degB, double);
-    Calloc(mu_sol, degB, double);
-
-    pol_sqr.mu = mu_sqr;
-    pol_sol.mu = mu_sol;
-
-    if (degB <= 0) {
-      printf("LanDos with degB <= 0 not yet implemented");
-      exit(-1);
-    } else {
-      lsPol(&intv[4], degB, isqrt, tau, &pol_sqr);
-      lsPol(&intv[4], degB, rec, tau, &pol_sol);
-    }
-  }
-
```

Comment:
these were removed, since it is part of the B-sol. LandosG should be orthogonal to the choice of Bsol, which was set outside and before this function is called.